desktop/com.github.xournalpp.xournalpp.desktop

```diff
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
```

Comment:
The Freedesktop specification depreciated the _Encoding_ key item which will be removed.